pypy-requirements.txt

```diff
@@ -1,60 +1 @@
-ConfigArgParse==0.10.0
```

Comment:
Currently no but since one more file doesn't matter, it's always possible we may need it later. We did have the differing readline dep recently which would have needed it